src/http-common.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios'
+
+export default axios.create({
+    baseURL: '',
```

Comment:
@alextakada322 can u edit this and add the base url for the backend please

Reply:
added baseURL heroku